<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gridGraph</class>
 <widget class="QMainWindow" name="gridGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gridGraph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#gridGraph {
background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_grid_graph_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#create_grid_graph_button{
	background-color: rgb(0, 250, 146);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
	color: black;
}

#create_grid_graph_button:pressed {
 background-color: red; 
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Create Grid Graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="M_size">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#M_size{
	background-color:  rgb(0, 250, 146);
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="N_size">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#N_size{
	background-color:  rgb(0, 250, 146);
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>M:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>13</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gravity_center">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#gravity_center{
	background-color:  rgb(0, 250, 146);
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>13</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Gravity Center:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_properties_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>45</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#add_properties_button {
	background-color: rgb(0, 250, 146);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
	color: black;
}

#add_properties_button:pressed {
 background-color: red; 
border-style: inset;
}

</string>
    </property>
    <property name="text">
     <string>Add Properties</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>13</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Initial Level Weight:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="initial_weight">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#initial_weight{
	background-color:  rgb(0, 250, 146);
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="show_mst_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>45</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#show_mst_button {
	background-color: rgb(0, 250, 146);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
	color: black;
}

#show_mst_button:pressed {
 background-color: red; 
border-style: inset;
}
</string>
    </property>
    <property name="text">
     <string>Show MST</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="total_weight">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>5</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#total_weight{
	background-color:  rgb(0, 250, 146);
	color: black;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
